src/network/api/footerSocialLinksApi.ts

```diff
@@ -0,0 +1,18 @@
+import NetworkCall from "../networkCall";
```

Comment:
Inconsistency in the filename. Isn't it should be tsx?

Reply:
No, it should be just ts because it's not containing html/jsx